src/js/components/ExpandableGroup/ExpandableGroup.tsx

```diff
@@ -65,6 +80,49 @@ const ExpandableGroup = ({
 
   const className = getClassName('expandable-group', error && 'u-padBottomSm', mods);
 
+  const deleteWithTooltip = (
+    <PopupTooltip text={deleteTooltip}>
+      <Icon
+        testId={`ExpandableGroup--delete-${testId}`}
+        mods={`${
+          deleteDisabled ? 'disabled u-colorNeutral5' : 'expandable-group--trash u-colorNegative4'
+        }`}
+        name="trash"
+        style={{
+          width: 24,
+          height: 24,
+          cursor: deleteDisabled ? 'not-allowed' : 'pointer',
+        }}
+      />
+    </PopupTooltip>
+  );
+
+  const deleteWithoutTooltip = (
+    <Icon
+      testId={`ExpandableGroup--delete-${testId}`}
+      mods={` ${deleteDisabled ? 'u-colorNeutral5' : 'expandable-group--trash u-colorNegative4'}`}
+      name="trash"
+      style={{
```

Comment:
Looks like we can pull this out to be something like

```JavaScript
const iconStyles = {
  width: 24,
  height: 24,
  cursor: deleteDisabled ? 'not-allowed' : 'pointer',
}
```

And then just have

```JavaScript
<Icon
  ...
  style={iconStyles}
/>
```

on Both `deleteWithTooltip` and `deleteWithoutTooltip`

For extra points you could actually simplify the Icon to it's own function like so...

```JavaScript
  const renderIcon = (deleteDisabled, testId, mods = '') => (
    <Icon
      testId={`ExpandableGroup--delete-${testId}`}
      mods={`${
        deleteDisabled ? `${mods} u-colorNeutral5` : 'expandable-group--trash u-colorNegative4'
      }`}
      name="trash"
      style={iconStyles}
    />
  )
```

and then call it like...

```JavaScript
  renderIcon(deleteDisabled, testId, 'disabled')
```

Reply:
I've got this refactored, see latest commit. I can just reuse the "without tooltip" portions. Great catch 👍